workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstanceStatusIndicator.tsx

```diff
@@ -43,16 +47,50 @@ export const WorkflowInstanceStatusIndicator = ({
     return VALUE_UNAVAILABLE;
   }
 
+  let icon: React.ReactNode;
+  let title: string = '';
+  switch (status) {
+    case ProcessInstanceStatusDTO.Active:
+      icon = <CircularProgress size="1.15rem" />;
```

Comment:
```suggestion
      icon = <CircularProgress size="1.15rem"  className={iconColor} />;
```
Maybe there is no obvious change but it will make the code coherent.

Reply:
it bothers me as well thanks :)